src/main/java/io/bf2/kafka/authorizer/CustomAclAuthorizer.java

```diff
@@ -227,6 +227,7 @@ public void configure(Map<String, ?> configs) {
         }
 
         configureDefaults(defaultBindings);
+        deleteOrphanBindingsFromDelegate();
```

Comment:
Should we add a feature flag to disable this or add a dry-run mode that would just list and log our affected bindings? A little nerve-wracking to imagine a bug destroying ZK configuration incorrectly.